### PR TITLE
Rearrange "+" buttons in list screens

### LINKED ISSUE #421: Task: Rearrange "+" buttons in list screens
- [x] Add text next to the "+" button in the header to make it stand out more (P1)
- [x] Remove "Add new device" button (P2)

Pages : 
- [x] Device List
- [x] Device Model List
- [x] IoT Edge List
- [x] Concentrator List

### PR DESCRIPTION
**Pull request by ChristopheRib63**
## Description

What's new?

- Fix #421

## What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

### CODE REVIEW

File: src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor [hunk 1]
```diff
@@ -37,7 +37,9 @@
                     <MudText Typo="Typo.h6">Device Models</MudText>
                     <MudSpacer />
                     <MudIconButton Icon="@Icons.Material.Filled.Refresh" Size="Size.Medium" OnClick="LoadDeviceModels" Class="ma-2"></MudIconButton>
-                    <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="AddDeviceModel" Class="ma-2"></MudIconButton>
+                    <MudItem xs="0">
+                        <a href="/device-models/new"><MudFab Color="Color.Secondary" Icon="@Icons.Material.Filled.Add" Label="new device-models" id="newDeviceButton" /></a>
```
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor:41]: Why using an HTML tag instead of the MudIconButton ?

**Comment by ChristopheRib63** [comment]
image

Voici les modifications pour un nouvel élément. Votre avis?

File: src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor [hunk 2]
```diff
@@ -37,7 +37,9 @@
                     <MudText Typo="Typo.h6">Device Models</MudText>
                     <MudSpacer />
                     <MudIconButton Icon="@Icons.Material.Filled.Refresh" Size="Size.Medium" OnClick="LoadDeviceModels" Class="ma-2"></MudIconButton>
-                    <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="AddDeviceModel" Class="ma-2"></MudIconButton>
+                    <MudItem xs="0">
```
[the remark this answers is not in the document]
Comment (ChristopheRib63) [reply src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor:40]: Position du  bouton par rapport au rebord de droite

**Comment by ChristopheRib63** [comment]
image

File: src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChristopheRib63) [reply src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor:41]: Modified

File: src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor [hunk 3]
```diff
@@ -37,7 +37,7 @@
                     <MudText Typo="Typo.h6">Device Models</MudText>
                     <MudSpacer />
                     <MudIconButton Icon="@Icons.Material.Filled.Refresh" Size="Size.Medium" OnClick="LoadDeviceModels" Class="ma-2"></MudIconButton>
-                    <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="AddDeviceModel" Class="ma-2"></MudIconButton>
+                           <a href="/device-models/new"><MudFab Color="Color.Secondary" Icon="@Icons.Material.Filled.Add" Size="Size.Medium"  id="newDeviceButton" /></a>
```
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor:40]: This moves the MudIcon button to a HTML tag.
Why ? is there other options to keep the MudTag instead ? and us the AddDeviceModel Event?

File: src/AzureIoTHub.Portal/Client/Pages/Devices/DeviceListPage.razor [hunk 4]
```diff
@@ -87,7 +87,7 @@
                 <MudText Typo="Typo.h6">Devices</MudText>
                 <MudSpacer />
                 <MudIconButton Icon="@Icons.Material.Filled.Refresh" OnClick=@Search Size="Size.Medium" id="tableRefreshButton" Class="ma-2"></MudIconButton>
-                <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="AddDevice" id="tableAddItemButton" Class="ma-2"></MudIconButton>
+                 <a href="/devices/new"><MudFab Color="Color.Secondary" Icon="@Icons.Material.Filled.Add" Size="Size.Medium"  id="newDeviceButton" /></a>
```
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal/Client/Pages/Devices/DeviceListPage.razor:90]: Same as previous

File: src/AzureIoTHub.Portal/Client/Pages/LoRaWAN/Concentrator/ConcentratorListPage.razor [hunk 5]
```diff
@@ -24,7 +24,7 @@
                 <MudText Typo="Typo.h6">LoRaWAN Concentrators</MudText>
                 <MudSpacer />
                 <MudIconButton Icon="@Icons.Material.Filled.Refresh" Size="Size.Medium" OnClick="Search" Class="ma-2"></MudIconButton>
-                <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="AddDevice" Class="ma-2"></MudIconButton>
+                       <a href="lorawan/concentrators/new"><MudFab Color="Color.Secondary" Icon="@Icons.Material.Filled.Add" Size="Size.Medium"  /></a>
```
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal/Client/Pages/LoRaWAN/Concentrator/ConcentratorListPage.razor:27]: Same as previous

File: src/AzureIoTHub.Portal/Client/Pages/Edge_Devices/EdgeDeviceListPage.razor [hunk 6]
```diff
@@ -67,7 +67,7 @@
                 <MudText Typo="Typo.h6">Gateways</MudText>
                 <MudSpacer />
                 <MudIconButton Icon="@Icons.Material.Filled.Refresh" Size="Size.Medium" OnClick="Search" Class="ma-2"></MudIconButton>
-                <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="ShowAddDialog" Class="ma-2"></MudIconButton>
+                      <MudFab Color="Color.Secondary" Icon="@Icons.Material.Filled.Add" Size="Size.Medium"  OnClick="ShowAddDialog" />
```
Comment (kbeaugrand) [inline src/AzureIoTHub.Portal/Client/Pages/Edge_Devices/EdgeDeviceListPage.razor:70]: Here you kept the Mud Tag...

**Review by kbeaugrand — CHANGES_REQUESTED** [review]

**Review by kbeaugrand — APPROVED** [review]

**Pull request merged**